pkg/selinux/consts.go

```diff
@@ -17,6 +17,6 @@
 package selinux
 
 const (
-	RTEContextType  = "rte.process"
+	RTEContextType  = "container_device_plugin_t"
```

Comment:
skimming over the linked image this it seems there's a nice overlap over the features RTE needs and the features granted to this context. This seems still somehow a slight abuse (RTE is most certainly not a device plugin) but it seems the best we can do with the default contexts

Reply:
Yes I agree, but RTE operate in a similar fashion to device plugin in the manner of communication with Kubelet, so I find this context type fair enough.